tests/arduino/arduino.ino

```diff
@@ -4,32 +4,50 @@
 MAKE SURE YOU HAVE SET '\n' LINE ENDINGS
 ***************************************/
 
+uint8_t led_state;
+
 void set_led(void)
 {
-    int status;
-    if (vic_args("%d", &status) == 1) {
-        digitalWrite(13, status);
+    char status[5];
+    if (vic_args("%3s", &status) == 1) {
+        if (strcmp(status, "on") == 0) {
+            led_state = 1;
+        } else if (strcmp(status, "off") == 0) {
+            led_state = 0;
+        }
     }
 }
 
 void led_on(void)
 {
-    digitalWrite(13, HIGH);
+    led_state = 1;
 }
 
 void led_off(void)
 {
-    digitalWrite(13, LOW);
+    led_state = 0;
+}
+
+void serial_print(char c)
+{
+    Serial.print(c);
 }
 
+float test = 0;
+
 void setup()
 {
     Serial.begin(115200);
     pinMode(13, OUTPUT);
 
+    vic_output_set(serial_print);
+
     vic_fn_add("on", led_on);
     vic_fn_add("off", led_off);
-    vic_fn_add("set", set_led);
+    vic_fn_add("led", set_led);
+
+    vic_var_set("state", "0");
+    vic_var_bind("state", &led_state, VIC_VAR_UINT8);
```

Comment:
@vargac I kind of understand that you get what this example arduino sketch will do right away but many people will not when they upload it for the first time.

Would it be possible to print a simple introductory message in `setup` that will let the user know how to interact with it?